Changes.md

```diff
@@ -14,6 +14,7 @@ API
 ---
 
 - Display : Added `driverClosed()` method.
+- SceneGadget : Add `snapshotToFile()` method.
```

Comment:
Add -> Added

Reply:
Fixed in https://github.com/GafferHQ/gaffer/pull/5533/commits/6f11f56809664346e00e959e968ce0992adc28ff